cmd/query/app/flags_test.go

```diff
@@ -60,6 +61,56 @@ func TestQueryBuilderFlags(t *testing.T) {
 	assert.Equal(t, 10*time.Second, qOpts.MaxClockSkewAdjust)
 }
 
+func TestQueryBuilderFlagsSeparatePorts(t *testing.T) {
+	v, command := config.Viperize(AddFlags)
+	command.ParseFlags([]string{
```

Comment:
Do you need all the flags for this test? I think only `--query.http-server.host-port` is under test here, right?